src/Util/JsonSerializer.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace App\Util;
+
+use JMS\Serializer\SerializationContext;
+use JMS\Serializer\SerializerInterface;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Serializes entities into Json.
+ */
+class JsonSerializer
+{
+    /**
+     * Instance of the JMS Serializer.
+     *
+     * @var SerializerInterface $serializer
+     */
+    private $serializer;
+
+    /**
+     * The json string, containing the serialized string.
+     *
+     * @var string
+     */
+    private $json = '{}';
```

Comment:
I'm not sure if this would work in the version of PHP we are in. 
`
```suggestion
    private string $json = '{}';
```

Reply:
No complaints from php 8+ when using 3v4l, but I like the type hint.